package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "website-scraper",
-  "version": "4.2.3",
+  "version": "5.0.0",
```

Comment:
I think we should bump version after we finish all required changes for v5

Reply:
I'm ok with it. I'll revert.